src/extensions/default/CodeFolding/main.js

```diff
@@ -284,15 +279,18 @@ define(function (require, exports, module) {
     }
 
     /**
-        Initialise the extension
-    */
+     * Initialise the extension
+     */
     function init() {
-        if (CodeMirror.fold.combine || !prefs.getSetting("enabled")) {
```

Comment:
Per discussion in the PR, it seems like `CodeMirror.fold.combine` should be needed in the extension-registry copy of this, but not in the core version since it's guaranteed to always load first.